src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library.cljs

```diff
@@ -25,17 +26,21 @@
 
 (defn- render-property [library-schema library-attributes property-key]
   [:div {:style {:display "flex" :padding "0.5em 0" :borderBottom (str "2px solid " (:line-default style/colors))}}
-   [:div {:style {:flexBasis "33%" :fontWeight "bold" :paddingRight "2em"}}
+   [:div {:style {:flexBasis "40%" :fontWeight "bold" :paddingRight "2em"}}
     (get-in library-schema [:properties property-key :title])]
-   [:div {:style {:flexBasis "67%"}}
-    (render-value (get library-attributes property-key))]])
+   [:div {:style {:flexBasis "60%"}}
+    (render-value (get library-attributes property-key))
+    (if (= (get-in library-schema [:properties property-key :typeahead]) "ontology")
+      ;; check if it's a doid ontology?
+      [:span {:style {:fontStyle "italic"}} (str " (" (common/url-to-doid (get library-attributes :library:diseaseOntologyDOID)) ")")])]])
```

Comment:
add a TODO comment: when we support multiple ontology-driven properties, can't hardcode the property name here.